File: Backend/Backend/Model/UserAccount.cs
using System;
using System.Collections.Generic;

namespace Backend.Model;

public partial class UserAccount
{
    public int UserId { get; set; }

    public string UserName { get; set; } = null!;

    public string? Password { get; set; }

    public string? UserProfile { get; set; }
}

File: Frontend/src/app/model/userAccount.ts
export class UserAccount{
    public userId: number;
    public userName: string;
    public password: string;
    public userProfile: string;

    constructor(userId: number, userName: string, password: string, userProfile:string){
        this.userId = userId;
        this.userName = userName;
        this.password = password;
        this.userProfile = userProfile;
    }
}